src/Graphics/OpenGLContext/opengl_GLInfo.cpp

```diff
@@ -61,7 +60,7 @@ void GLInfo::init() {
 	if (isGLESX && bufferStorage)
 		g_glBufferStorage = (PFNGLBUFFERSTORAGEPROC) eglGetProcAddress("glBufferStorageEXT");
 #endif
-	texStorage = (isGLESX && (numericVersion >= 30)) || (!isGLESX && numericVersion >= 42) ||
+	texStorage = (isGLESX && (numericVersion >= 30) && config.video.multisampling != 0) || (!isGLESX && numericVersion >= 42) ||
```

Comment:
If I remember correctly, it was because text storage wasn't being used when multisampling was enabled. But I'll have to double check. It must have caused me some issue at some point and I'm not remembering everything that must had forced me to add this.